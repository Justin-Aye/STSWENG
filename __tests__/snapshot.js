// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="text-center mt-5"
  >
    <div
      class="mb-5 w-1/4 mx-auto bg-[#4487d4] rounded-lg py-2 cursor-pointer hover:brightness-90"
    >
      <p
        class="text-[20px] text-white"
      >
        POST AN IMAGE
      </p>
    </div>
    <div
      class="w-[50px] h-[50px] mx-auto mb-5 relative justify-center"
    >
      <img
        alt=""
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="(max-width: 500px)"
        src="/_next/image?url=%2Fimages%2Floading.gif&w=3840&q=75"
        srcset="/_next/image?url=%2Fimages%2Floading.gif&w=16&q=75 16w, /_next/image?url=%2Fimages%2Floading.gif&w=32&q=75 32w, /_next/image?url=%2Fimages%2Floading.gif&w=48&q=75 48w, /_next/image?url=%2Fimages%2Floading.gif&w=64&q=75 64w, /_next/image?url=%2Fimages%2Floading.gif&w=96&q=75 96w, /_next/image?url=%2Fimages%2Floading.gif&w=128&q=75 128w, /_next/image?url=%2Fimages%2Floading.gif&w=256&q=75 256w, /_next/image?url=%2Fimages%2Floading.gif&w=384&q=75 384w, /_next/image?url=%2Fimages%2Floading.gif&w=640&q=75 640w, /_next/image?url=%2Fimages%2Floading.gif&w=750&q=75 750w, /_next/image?url=%2Fimages%2Floading.gif&w=828&q=75 828w, /_next/image?url=%2Fimages%2Floading.gif&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Floading.gif&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Floading.gif&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Floading.gif&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Floading.gif&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
  </div>
</div>
`;