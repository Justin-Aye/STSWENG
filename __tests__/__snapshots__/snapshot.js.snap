// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="text-center mt-0 flex flex-col"
  >
    <div
      class="bg-doc_bg w-full self-center pt-8"
    >
      <div
        class="mb-5 w-2/5 mx-auto bg-nav_bg rounded-full py-2 px-5 cursor-pointer hover:transition duration-300 hover:bg-nav_bg_dark flex justify-center items-center"
      >
        <img
          alt=""
          data-nimg="1"
          decoding="async"
          height="51"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fadd_image_icon_w.png&w=128&q=75"
          srcset="/_next/image?url=%2Fimages%2Fadd_image_icon_w.png&w=64&q=75 1x, /_next/image?url=%2Fimages%2Fadd_image_icon_w.png&w=128&q=75 2x"
          style="color: transparent;"
          width="60"
        />
        <span
          class="text-[20px] text-white w-fit h-fit"
        >
          Create New Post
        </span>
      </div>
    </div>
    <div
      class="w-[50px] h-[50px] mx-auto mb-5 relative justify-center"
    >
      <img
        alt=""
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="(max-width: 500px)"
        src="/_next/image?url=%2Fimages%2Floading.gif&w=3840&q=75"
        srcset="/_next/image?url=%2Fimages%2Floading.gif&w=16&q=75 16w, /_next/image?url=%2Fimages%2Floading.gif&w=32&q=75 32w, /_next/image?url=%2Fimages%2Floading.gif&w=48&q=75 48w, /_next/image?url=%2Fimages%2Floading.gif&w=64&q=75 64w, /_next/image?url=%2Fimages%2Floading.gif&w=96&q=75 96w, /_next/image?url=%2Fimages%2Floading.gif&w=128&q=75 128w, /_next/image?url=%2Fimages%2Floading.gif&w=256&q=75 256w, /_next/image?url=%2Fimages%2Floading.gif&w=384&q=75 384w, /_next/image?url=%2Fimages%2Floading.gif&w=640&q=75 640w, /_next/image?url=%2Fimages%2Floading.gif&w=750&q=75 750w, /_next/image?url=%2Fimages%2Floading.gif&w=828&q=75 828w, /_next/image?url=%2Fimages%2Floading.gif&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Floading.gif&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Floading.gif&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Floading.gif&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Floading.gif&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
  </div>
</div>
`;
